crates/revm/src/builder.rs

```diff
@@ -195,6 +195,36 @@ impl<'a, EXT, DB: Database> EvmBuilder<'a, HandlerStage, EXT, DB> {
         }
     }
 
+    /// This modifies the [EvmBuilder] to make it easy to construct an [`Evm`] with a _specific_
+    /// handler.
+    ///
+    /// # Example
+    /// ```rust
+    /// use revm::{EvmBuilder, Handler, primitives::{SpecId, HandlerCfg}};
+    /// use revm_interpreter::primitives::CancunSpec;
+    /// let builder = EvmBuilder::default();
+    /// // set up empty handler cfg
+    /// let handler = HandlerCfg::new(SpecId::LATEST);
+    /// let builder = builder.with_handler_cfg(handler);
```

Comment:
Would remove this lines, they can be confusing
```suggestion
```

Reply:
Second comment will address why you needed this previously: https://github.com/bluealloy/revm/pull/1124#discussion_r1501291862